core/meta/CMakeLists.txt

```diff
@@ -87,7 +87,24 @@ ROOT_OBJECT_LIBRARY(Meta
   src/TVirtualStreamerInfo.cxx
 )
 
-target_include_directories(Meta PRIVATE res ../foundation/res)
+target_include_directories(Meta PRIVATE
+  ${CMAKE_SOURCE_DIR}/core/meta/res
```

Comment:
We should use relative directories for all of those, to enable relocation.

Reply:
I tried relative paths, but this leads even to more problems. 

cmake is not able to detect that directory "/home/user/root/core/base/../cling" and "/home/user/root/core/gui/../cling" are the same. As result - in C flags too many includes paths like that are populated.